doc/api/cli.md

```diff
@@ -1257,6 +1257,19 @@ Print node's version.
 
 ## Environment variables
 
+### `FORCE_COLOR=[1, 2, 3]`
+
+The `FORCE_COLOR` environment variable was first introduced by modules in
+the Node.js ecosystem before being introduced into Node.js. It is used to
+enable ANSI colorized output. The value may be:
```

Comment:
Optional nit: I don't think the first sentence is of interest for documentation purposes. Or at least it should not be the first sentence. The first sentence should tell us what the thing does.

```suggestion
The `FORCE_COLOR` environment variable is used to
enable ANSI colorized output. The value may be:
```